pkg/snapshot/restorer/restorer_suite_test.go

```diff
@@ -101,6 +102,21 @@ func cleanUp() {
 
 	err = os.RemoveAll(snapstoreDir)
 	Expect(err).ShouldNot(HaveOccurred())
+
+	//for the negative scenario for invalid restoredir set to "" we need to cleanup the member folder in the working directory
+	restoreDir := path.Clean("")
+	info, err := os.Stat(fmt.Sprintf("%s/%s", restoreDir, "member"))
```

Comment:
No need to check for existence of the directory. You can simply run `os.RemoveAll`. If the directory doesn't exist, it'll return nil error anyway.

Also, use `path.Join(restoreDir, "member")` for joining `restoreDir` and `member`.